test/dotnet-test.yml

```
@@ -0,0 +1,44 @@
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint-test-coverage:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)